components/third_party/adblock/README.chromium

```diff
@@ -1,3 +1,3 @@
 Name: Brave Ad Block
-URL: https://github.com/brave/brave-core-crx-packager
```

Comment:
This name (adblock) was a reference to the C++ adblock library.  Should this whole section / file be updated to point to adblock-rs?

Reply:
It previously pointed to the CRX packager with the license set to "various", so I figured it is referring to our collection of filter lists as a whole - which I think `adblock-resources` is most representative of for now

Reply:
okie dokie :)